gpu/install_gpu_drivers.sh

```diff
@@ -87,23 +96,23 @@ function validate_gpu_updated() {
     done
 
     if [ $count == 50 ]; then
-        echo "################## Failed to install gpu ####################"
+        echo "################## Failed to integrate with gpu ####################"
```

Comment:
nit: shouldn't text match the above

Reply:
actually this part is more about verifying integration ... rather than driver installation